src/crimson/os/cyanstore/cyan_store.cc

```diff
@@ -647,4 +661,76 @@ unsigned CyanStore::get_max_attr_name_length() const
   // arbitrary limitation exactly like in the case of MemStore.
   return 256;
 }
+
+seastar::future<FuturizedStore::OmapIterator> CyanStore::get_omap_iterator(
+    CollectionRef ch,
+    const ghobject_t& oid
+  ) {
+  auto c = static_cast<Collection*>(ch.get());
+  auto o = c->get_object(oid);
+  if (!o)
+    throw std::runtime_error(fmt::format("object does not exist: {}", oid));
+  return seastar::make_ready_future<FuturizedStore::OmapIterator>(CyanStore::CyanOmapIterator(o));
+}
+
+seastar::future<std::map<uint64_t, uint64_t>>
+CyanStore::fiemap(
+    CollectionRef ch,
+    const ghobject_t& oid,
+    uint64_t off,
+    uint64_t len
+  ) {
+  std::map<uint64_t, uint64_t> m;
+  auto c = static_cast<Collection*>(ch.get());
+
+  ObjectRef o = c->get_object(oid);
+  if (!o)
+    throw std::runtime_error(fmt::format("object does not exist: {}", oid));
+  m[0] = o->get_size();
+  return seastar::make_ready_future<std::map<uint64_t, uint64_t>>(std::move(m));
```

Comment:
```suggestion
  std::map<uint64_t, uint64_t> m{{0, o->get_size()}};
  return seastar::make_ready_future<std::map<uint64_t, uint64_t>>(std::move(m));
```

for 2 reasons, 1) define the variable close to where it's used, 2) use initialiser list for more concise code.